src/components/publiq-ui/Panel.js

```diff
@@ -1,27 +1,61 @@
 import PropTypes from 'prop-types';
 import { getValueFromTheme } from './theme';
 import { Stack, stackPropTypes, getStackProps } from './Stack';
+import { Children } from 'react';
 
 const getValue = getValueFromTheme('panel');
 
-const Panel = ({ children, className, ...props }) => (
-  <Stack
-    css={`
-      border: 1px solid ${getValue('borderColor')};
-      box-shadow: 0 1px 1px rgba(0, 0, 0, 0.05);
-    `}
-    {...getStackProps(props)}
-    className={className}
-    marginBottom={4}
-  >
-    {children}
-  </Stack>
-);
+const Panel = ({ children, className, ...props }) => {
+  const parsedChildren =
+    Children.count(children) === 1 ? <>{children}</> : children;
```

Comment:
I see this line is added in several components. But why is this needed?

Reply:
It's a fix for passing a single element to Panel and ListItem (these are Inlines now).